src/FTPC.jl

```diff
@@ -1,6 +1,8 @@
+using Compat
```

Comment:
You should set the minimum required version of Compat to v0.41 (in the REQUIRE file). This is the first version of Compat that includes `Nothing` and `Cvoid`.